changelogs/CHANGELOG-v0.3.1.md

```diff
@@ -0,0 +1,69 @@
+The v0.3.1 release is a patch release which includes some bug fixes and makes
+the ability for bpf enabled applications to receive their maps via bpfd's
+custom CSI plugin default.
+
+This enablement means that most applications can now access their maps **WITHOUT**
+being run as root.
+
+> **WARNING**
+The CSI feature still requires a privileged application on distributions which
+enable seLinux by default (i.e openshift).  Therefore we've shipped a set of
+deployment configs specifically for openshift in this release, see the additional
+`go-<example>-counter-install-ocp-v0.3.1.yaml`` artifacts included in the release
+payload. Stay tuned to [#829](https://github.com/bpfd-dev/bpfd/issues/829) for
+updates.
+
+The new yaml syntax that should be used by bpf enabled applications resembles
+the following:
+
+```yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: <APP_NAME>
+  ...
+spec:
+  ...
+  template:
+    ...
+    spec:
+     ...
+          volumeMounts:
+            - name: bpf-maps
+              mountPath: /bpf-maps
+      volumes:
+        - name: bpf-maps
+          csi:
+            driver: csi.bpfd.dev
+            volumeAttributes:
+              csi.bpfd.dev/program: <*Program_Name>
+              csi.bpfd.dev/maps: <BPF Map Names>
+```
+
+Additionally, this release removes all dependencies involved with deploying bpfd
+with TLS, meaning cert-manager deps are completely removed from the operator
+therefore simplifying the deployment considerably.
+
```

Comment:
Do we need to highlight that the bpfd user/group was removed?

Reply:
I'm not sure.... I guess it is sort of a user facing change if anyone was running bpfd with systemd previously? 

So yeah we should

Reply:
Done